### ISSUE TITLE
Use raw swagger file instead of embed it in a go file

### ISSUE THREAD

**Issue by thoriqsatriya** [swaggo/files#1]
Don't you think it's better if the swagger files are the original swagger files, instead of embedded into go files? 
And I think it's better if we can specify where to look the swagger file (specify the swagger-file directory path in the project).

The reason is sometimes we need to modify some of swagger files, for example theme color, icon and title.
What do you think?

Thank you

**Comment by easonlin404** [comment]
I agree with you It's more flexible if you want to modified the swagger files. We will add new method that can specify swagger files directory path.

**Comment by bluntdel** [comment]
How is it going ? @easonlin404 
We need add some params like "sn" to all my post request,so we need modify swagger.js
How can we realize it ?
